packages/redux/src/promotionEvaluations/actions/__tests__/fetchPromotionEvaluationItems.test.ts

```diff
@@ -0,0 +1,91 @@
+import { actionTypes } from '../..';
+import { fetchPromotionEvaluationItems } from '..';
+import { getPromotionEvaluationItems } from '@farfetch/blackout-client/promotionEvaluations';
+import { INITIAL_STATE } from '../../reducer';
+import {
+  mockPromotionEvaluationId,
+  mockPromotionEvaluationsItemsResponse,
+} from 'tests/__fixtures__/promotionEvaluations';
+import { mockStore } from '../../../../tests';
+import find from 'lodash/find';
+
+jest.mock('@farfetch/blackout-client/promotionEvaluations', () => ({
+  ...jest.requireActual('@farfetch/blackout-client/promotionEvaluations'),
+  getPromotionEvaluationItems: jest.fn(),
+}));
+
+const promotionEvaluationsMockStore = (state = {}) =>
+  mockStore({ promotionEvaluations: INITIAL_STATE }, state);
+const expectedConfig = undefined;
+let store;
+
+describe('fetchPromotionEvaluationItems() action creator', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    store = promotionEvaluationsMockStore();
+  });
+
+  it('should create the correct actions for when the fetch promotion evaluation items procedure fails', async () => {
+    const expectedError = new Error('Fetch promotion evaluation items error');
+
+    getPromotionEvaluationItems.mockRejectedValueOnce(expectedError);
+
+    expect.assertions(4);
+
+    try {
+      await store.dispatch(
+        fetchPromotionEvaluationItems(mockPromotionEvaluationId),
+      );
+    } catch (error) {
+      expect(error).toBe(expectedError);
+      expect(getPromotionEvaluationItems).toHaveBeenCalledTimes(1);
+      expect(getPromotionEvaluationItems).toHaveBeenCalledWith(
+        mockPromotionEvaluationId,
+        expectedConfig,
+      );
+      expect(store.getActions()).toMatchObject([
```

Comment:
At least in `next` we're avoiding `toMatchObject` and we're using `toEqual`, to avoid some false positives we had in the past (a lot). And with that, we no longer need to have any snapshot (like in the test below).

You can check (almost) every other e.commerce action to see what I mean. Let me know your thoughts! 🙂

Reply:
I agree and I had a false positive indeed 😄  Fixed 🙂